tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/controller.go

```diff
@@ -69,30 +70,117 @@ func load(ctx context.Context, kubeClientSet kubernetes.Interface, o *cl.ObjectS
 	return nil
 }
 
+func loadCacheConfig(ctx context.Context, kubeClientSet kubernetes.Interface, p *db.ConnectionParams) (bool, error) {
+	configMap, err := kubeClientSet.CoreV1().ConfigMaps(system.Namespace()).Get(ctx,
+		"cache-config", metaV1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+	if configMap.Data["disabled"] == "true" {
+		return true, nil
+	}
+	p.DbName = configMap.Data["dbName"]
+	p.DbDriver = configMap.Data["driver"]
+	p.DbHost = configMap.Data["host"]
+	p.DbPort = configMap.Data["port"]
+	p.DbExtraParams = configMap.Data["extraParams"]
+	p.DbUser = configMap.Data["user"]
+	p.DbPwd = configMap.Data["password"]
+	timeout, err := time.ParseDuration(configMap.Data["timeout"])
+	if err != nil {
+		return false, err
+	}
+	p.Timeout = timeout
+	return false, nil
+}
+
 var params db.ConnectionParams
 
+func initCache(ctx context.Context, kubeClientSet kubernetes.Interface, params db.ConnectionParams) *taskCache.TaskCacheStore {
+	logger := logging.FromContext(ctx)
+	disabled, err := loadCacheConfig(ctx, kubeClientSet, &params)
+	if err != nil {
+		logger.Errorf("Config map could not be loaded. Error : %w", err)
+		params.LoadMySQLDefaults()
+		params.Timeout = 10 * time.Second
+	}
+	cacheStore := &taskCache.TaskCacheStore{Params: params}
+	if disabled || strings.EqualFold(os.Getenv("CACHE_STORE_DISABLED"), "true") {
```

Comment:
since now we can configure the cache disable flag via configmap, we can get rid of the `CACHE_STORE_DISABLED` env variable flag.

Reply:
```suggestion
	if disabled {
```